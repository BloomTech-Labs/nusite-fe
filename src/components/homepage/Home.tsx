import React from 'react';
import client from './images/client.jpg';
import userflow from './images/userflow.gif';
import Footer from './Footer';
import Header from './Header';

import './Homepage.css';

const Home = () => {
    
return (
    <>
    <Header />    
        <main className="section-main">
        <br />
        <br />
        <br />
        <h1 className="tagline">Connecting web developers to people with great ideas. Old website, new website, app, or feature, 
            we've got you covered.
        </h1>
        <br />
        <p>We are creating a marketplace for developers and project owners/clients to 
            meet each other and form partnerships. Developers and project owners can offer 
            their services and projects to one another and be matched according to their needs.  
        </p>
        <br />
        <br />
        </main>
        <div className="parallax">
            <br />
            <br />
            <h2 className="tagline-2">Comprehensive recommendation based on advanced data analysis.</h2>
        </div>
        <br />
        <section>
            <br />
            <h2>Project Builders</h2>
            <br />
            <img src={client} className="avatars" alt="" />
            <br />
            <p>A marketplace for developers and project owners to 
            meet each other and form partnerships. Developers and project owners can offer 
            their services and projects to one another and be matched according to their needs.  
            </p>
            <br />
            <br />
            <br />
            <div className="gif-wrapper">
            <img src={userflow} className="user-flow" alt="" />
            </div>
            <br />
            <br />
            <br />
            <p>
            Find someone to build, fix, or add on to your existing site or idea in minutes. Just set up a profile 
            and browse existing projects to get and idea of what you like, pick someone, and/or just 
            let us give your best matches for your project based on developer skills. Take the workload off of yourself 
            and get started today. It's easy.</p>
        <br />
        <br />
        <div className="google-btn">
        <div className="google-icon-wrapper">
        <img className="google-icon" src="https://upload.wikimedia.org/wikipedia/commons/5/53/Google_%22G%22_Logo.svg" alt="" />
        </div>
         <p className="btn-text"><b>Sign in with google</b></p>
        </div>
        <br />
        <div className="github-btn">
        <div className="github-icon-wrapper">
        <img className="github-icon" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAOEAAADhCAMAAAAJbSJIAAAAh1BMVEX///8YFhYAAAAQDQ0UEhJEQ0MKBgYTERHz8/P6+vpiYWFUU1O0tLQeHBwRDg4qKSmjoqLi4uLT09NJSEhpaGglIyPHxsaYl5eAgIA+PT14d3eJiIggHh7Nzc3d3d3U1NQxMDDr6+uura2Uk5NubW0wLi6EhIS7u7tjYmI4NzdOTU2WlpZaWVlxXGWoAAAIFElEQVR4nO2daZubvA6GB5lxICvZE5JZskynaef//75DdsAWEGLZ9D26P/VqU8wTW5Ys2c7LC8MwDMMwDMMwDMMwDMMwDMMwDMMwDMMwDMMwDMP8HxO6fgEaglF3vmiPD4PpureeDg7j9mLe/Qhcv5YZgtm8DSek6Lf8I62+kOe/+tnN/nGZb6vlUZrv6fGPQperN9evWZdZnLy/QMTdEcmn9jPXL/s474ukf7C+U/oyEbn4cP3KD9EdV5d3E/nadf3aVQkjUWFwqiT/K/onfEmi77HuS3ekjFy/fildWVvfRWOzx+po+ZS+s8blyLUMlDCuZX95BMQNNceZBAP6jgA00j8uoGVI4HGoxq7lKAwHpjrwDEyHriVl6RqxwDQCGjWprp6eQlV8WLmWdadtdoRegbZrYRfCVxqBicRxI9xG8EUlMJH42YAFcjCVZAI9T06dS6QVmEgcOJYYftEKTCR+urVFsknmDoxdCvxNLzCROHEncGVDYCJx50rgtx2BicRvNwKHSqjmn9K8zywxzk9Qn+smDB/kg21/OnubbRdfVbKkOvoAvThKnjHNSxSfLgQulDF6tZfNdgyPD2AJX/NLV6n27SIKn6ka4J7Ufd8/qFFC557c/9A823ryJpSKufnr9Ac2MWiCgVNlRv1rAe339H/uKZ9p9ezq041RT+b81rB9m3R8AXAtQMnrH8VVRgvGuZT+b/XLsT1ONePIAyWbO5vCub7U66yi2Wh4DTGD4WgWrTq9c10KPMUZRLrHv+c/RcqrZrYETZVsAetJNMKC52AUTdawUP/hTaNQdowqKKGrm0a0TmtT+izdJ4ba59tMMWrTMmBuERDqFLYGxp5fis5MEoUGW9A3YC37FgptZo1cob1O1HchvUJ7EXhLnxw1qFBrh0lgsDTXRBHYognK582qaOdST++QCND5QsPN6/zhkXzYRMM7FlKrMU1t5mgbNjJvv7DWDQYdHWyBafBbxMGLMMYMcYOuvMSroSYK0KwLbwpNhVVFbdDnM2K0dfhlrBHUDm0MU3SQSpMDaIJJpHeJ2Dzu+QbdYeLy11jGjnw23aHz+NxoO6gpkkduS2QeN55H6SAlH92C2SQB+tVuDbeky5Mc6RMbIjZ4fGG8qTEWHNIa4l8sJDafCdNmSjzy6LuNWYf5RFiA+CVij4i0SrL6RoJT2g1h2ERjMJy5gwQ2tFMNNsGROCksuDCZLVFAl/cU8TC2wjCYtVTBImKarxVrjDK9r6nHHPFpCkNTfWxKWmhDnAXRuhTx+aSRKdKm/CFp7Uf/fRoPENMc+nqFv0laQ0YMqcsf6C3Dch+aXadlUTZJnBE0pT0sqPlL0tqJUK2vnyAKM5Z6myBVqNtl4OX3KBhjjYTehKMU60PLHp/SDpFvlSaQQgpQTuZSmkAKLUA58Ic0mwiwjAlpTIMW1ihMAyk10+7JwJIYJEHN3l7G5A5WWSNxF6jRm0yu58Erl+aXwGiJzckan2IG3yJttWjCiwtYnsYTf4y3hcTdniQ974Um9c0bB14/oN2Gqd8N5RGcGMB8BXXxaYIdAvJN2z9eQKTdZ1pQfjbr9LGqhecL2nNQaAnY86XRmhDmDMk30uJTjdl6Albjol3/nviDHxYxGC9iywrPwrEE3BCTycbYOP1C1jAEM5oC6vMT5MFQIwVnw4kSl2n6BSfvwcwsgG9KsrITGildXNo3cby8SKCNzYmjwvNMMHg2egs7RQ3QBqUXkIrQBfFkFmXUK/4Gbez0RndFnfGhU3+tGJRcAeNLGwe7s76qJUHmpnYJcT1jCf+WHemj3hB1Ib1wg+lktT/kz7UC7B+/X264Kr/jzdJp2VSW77KzfDPPf/kSPnePrAGGUZU77Kwd7vq8Dctb0Slc5N+vDyDibRWVw+5iXe3osLWzXalszT3c/la74HQwexxHXfR0XjeKx9WPf9s6UPKS7kQPllfT+NCOsuPpUXz5P0mdJK3QhfaO5adTbuKWZkPWjoV5lU80xFax2IUvL4e01dymcO26QxSG48URUu6rsnmTa7a7brvadBuIS94LqxOoWAnY7mT30l8jNc2BoX7Jiqp6J1q+OWKTnVauravjtDQZjubU8g+yfatCNp15O2uhrHxKc38Vb4DpT6kVKSwzBnRbmMb5AK4sVC44AZR5jv0Lo7N3t9wPI2xT0Vfi8kuvQMJPcWUEUmzRLSNbG7pnF4LV9VoIaMUVEmNVFAqLB9VTZPJF/a/Uv3xEq1+r3Xe1yQ/N/d5xdQNPmLmCp7ahIDuf9APEMhlTFHUvHcN2P6QEujDCM5l5sG4ZuFQh0Ox8rEbGK9Yse5UplAenl+6l/XXN+aBEofAdX5y4T0ls1ZptihWKPuXekkrsM71Yo9ZdqFA48hMZ9hlbXD88sRcpFF4DBOai7Rb4i9QvqwzLc0cFCmXP+RA9k82Cn/JPr5M4nvzpAZQX3HCFsGzENcJHtvkklH+8s0wKv8pZE1Qh0JcKqzPS3T53MqTaCht1nXfCBrmMtrZC2bwfD1CS3k8phNeGzDFpZn1NN9ZTKEi349cnmKjZ+ToKfVjavUDwAWZKAbeGQrByy05tdrky28MKa5dXrbGZZBzHgwrlMyVya7y3U/34kEKAn3/kp8mOV+xeXEeF3MalaN4C2Dd2glEJdv55sFbYsHjcSZrEsoN5w+1PYXb6XUe/guNeJSHsorm/YlVA8B1VK9qGDYxfGIZhGIZhGIZhGIZhGIZhGIZhGIZhGIZhGIZhGIZhmP8q/wOa7F1sPxWUkQAAAABJRU5ErkJggg==" alt="" />
        </div>
        <a href="https://github.com/">
         <p className="btn-text"><b>Sign in with github</b></p>
        </a>
        </div>
        <br />
        <hr />
        <br />
        <h4 className="tagline">
        PartNerdâ„¢ provides users a comprehensive recommendation generation based on advanced data analysis.
            </h4>
        <br />
        <div className="container">
        <div className="button-big">
            <label><div className="button-small"><h3>Get Started</h3></div></label>
            <div className="bg"></div>
        </div>
        <br />
        </div>
        <hr />
        <br />
        <h2>For Developers </h2>
            <br />
            <p>
            Register as a developer, set up a brief profile, show off your skills and portfolio as they
            continue to grow, and get paid projects now. It's that simple, 
            no hiring managers, pick projects and 
            let the projects you want get matched with you today. </p>
            <br />
            <br />
        </section>
        <section className="center">
        <div id="social-test">
        <div className="social">
        <div>
        <li><i className="fa fa-facebook" aria-hidden="true"></i></li>
        <li><i className="fa fa-twitter" aria-hidden="true"></i></li>
        <li><i className="fa fa-github" aria-hidden="true"></i></li>
        <li><i className="fa fa-linkedin-square" aria-hidden="true"></i></li>
        </div>
        </div>
        </div>
        </section>
        <Footer />
        </>
    )
}

export default Home;
