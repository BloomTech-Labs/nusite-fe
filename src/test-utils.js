import React from "react";
import { render } from "@testing-library/react";
import { ThemeContext } from "./context/contexts";
import UserContext, { ANONYMOUS_USER } from "./context/user/context";

function MutationObserver(callback) {
   this.observe = jest.fn();
   this.disconnect = jest.fn();
   this.takeRecords = jest.fn();
}
global.MutationObserver = jest.fn(MutationObserver);

const mockSpies = {
   setDarkMode: jest.fn(),
   userDispatch: jest.fn(),
};

const AllTheProviders = ({ children }) => (
   <ThemeContext.Provider
      value={{ darkMode: false, setDarkMode: mockSpies.setDarkMode }}
   >
      <UserContext.Provider
         value={{
            userData: ANONYMOUS_USER,
            userDispatch: mockSpies.userDispatch,
         }}
      >
         {children}
      </UserContext.Provider>
   </ThemeContext.Provider>
);

const customRender = (ui, options) => {
   return {
      ...render(ui, { wrapper: AllTheProviders, ...options }),
      mockSpies,
   };
};

const testLinkArray = (linkArray, expected_length, expected_refs) => {
   expect(linkArray.length).toBe(expected_length);
   linkArray.forEach(element => {
      expect(expected_refs).toContain(element.getAttribute("href"));
   });
};

export * from "@testing-library/react";
export { customRender as render, testLinkArray };
