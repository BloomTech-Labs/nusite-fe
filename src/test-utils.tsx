import React from "react";
import { render, RenderResult } from "@testing-library/react";

import { ThemeContext } from "./context/contexts";

interface SpiesMap {
   setDarkMode: jest.Mock;
}

interface RenderWithSpiesResult extends RenderResult {
   mockSpies: SpiesMap;
}

const mockSpies: SpiesMap = {
   setDarkMode: jest.fn(),
};

const AllTheProviders: React.FC = ({ children }) => (
   <ThemeContext.Provider
      value={{ darkMode: false, setDarkMode: mockSpies.setDarkMode }}
   >
      {children}
   </ThemeContext.Provider>
);

const customRender = (ui: React.ReactElement, options?: Object | undefined) => {
   return {
      ...render(ui, { wrapper: AllTheProviders, ...options }),
      mockSpies,
   };
};

export * from "@testing-library/react";
export { customRender as render, RenderWithSpiesResult };
