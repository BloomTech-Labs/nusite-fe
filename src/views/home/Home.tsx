import React, { lazy, Suspense } from "react";
import Grid from "@material-ui/core/Grid";
import AboutSection from "./AboutSection";
import ProjectOwnersSection from "./ProjectOnwersSection";
import DevelopersSections from "./DevelopersSection";
import MatchingSection from "./MatchingSection";
import Parallax from "./Parallax";
const Hero = lazy(() => import("./Hero"));

const Home = (props: any) => {
   return (
      <>
         <Suspense fallback={<></>}>
            <Grid container justify="center">
               <Hero className="hero" />
               <Grid container className="main-container" direction="column">
                  <AboutSection />
                  <ProjectOwnersSection />
                  <Parallax />
                  <DevelopersSections />
                  <MatchingSection />
               </Grid>
            </Grid>
         </Suspense>
      </>
   );
};

export default Home;
