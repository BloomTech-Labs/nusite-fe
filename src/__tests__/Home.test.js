import "@testing-library/jest-dom/extend-expect";
import { MemoryRouter } from "react-router-dom";

import React from "react";
import { render } from "../test-utils";

import Home from "../views/home/Home";

describe("Home component test", () => {
   test("It displays all Home children components correctly without error.", () => {
      const home = render(
         <MemoryRouter>
            <Home />
         </MemoryRouter>
      );
      expect(home).toMatchSnapshot();
   });
});
