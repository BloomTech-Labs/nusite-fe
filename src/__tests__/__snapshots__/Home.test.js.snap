// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component test It displays all Home children components correctly without error. 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-justify-xs-center"
        style="display: none;"
      >
        <div
          class="MuiGrid-root main-container MuiGrid-container MuiGrid-direction-xs-column"
        >
          <main
            class="MuiGrid-root item-container MuiGrid-item"
          >
            <div
              class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3"
            >
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-md-9 MuiGrid-grid-lg-8"
              >
                <h1
                  class="MuiTypography-root MuiTypography-h1"
                >
                  About Us
                </h1>
                <p
                  class="MuiTypography-root about MuiTypography-body1"
                >
                  Welcome to PartNerd. We are a marketplace where clients and developers can form partnerships. Take the guess work out of finding a qualified developer for your existing project or next big idea and let us match you with a developer according to your needs. Our market place empowers clients to create a listing for their new or existing project and for developers to showcase their services to prospective clients.
                </p>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-md-3 MuiGrid-grid-lg-4"
              >
                <img
                  alt=""
                  class="makeStyles-developersImage-104"
                  src="developers.svg"
                />
              </div>
            </div>
          </main>
          <section
            class="MuiGrid-root MuiGrid-item"
          >
            <div
              class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3 MuiGrid-direction-xs-column-reverse"
            >
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-md-3 MuiGrid-grid-lg-4"
              >
                <img
                  alt=""
                  class="makeStyles-image-135"
                  src="reatime2.svg"
                />
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-md-9 MuiGrid-grid-lg-8"
              >
                <h1
                  class="MuiTypography-root MuiTypography-h1"
                >
                  Project Owners
                </h1>
                <p
                  class="MuiTypography-root MuiTypography-body1"
                >
                  Find someone to build, fix, or add on to your existing site or idea. Just create an account and start searching for qualified developers in seconds. You can browse through developer profiles to to see if a developer is suited for your project or just let our algorithm do the heavy lifting for you and let us find you the the best matches based on developer skills or other preferences of your project. With Partnerd it's easy to find a developer for your project in no time.
                </p>
                <div
                  class="MuiGrid-root MuiGrid-container MuiGrid-justify-xs-center"
                >
                  <a
                    aria-disabled="false"
                    class="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-button-136 MuiButton-containedPrimary"
                    href="/register"
                    role="button"
                    tabindex="0"
                  >
                    <span
                      class="MuiButton-label"
                    >
                      <div
                        class="makeStyles-buttonText-137"
                      >
                        Get Started
                      </div>
                    </span>
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </a>
                </div>
              </div>
            </div>
          </section>
          <div
            class="MuiGrid-root makeStyles-parallaxContainer-170 MuiGrid-item"
          >
            <div
              class="MuiGrid-root makeStyles-parallaxBrackround-171 MuiGrid-container MuiGrid-align-content-xs-center"
            >
              <span
                class="makeStyles-typingText-172"
              >
                <span />
                <span
                  class="typed-cursor"
                >
                  |
                </span>
              </span>
            </div>
          </div>
          <section
            class="MuiGrid-root item-container MuiGrid-item"
          >
            <div
              class="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column"
            >
              <div
                class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3"
              >
                <div
                  class="MuiGrid-root MuiGrid-item MuiGrid-grid-md-9 MuiGrid-grid-lg-8"
                >
                  <h1
                    class="MuiTypography-root MuiTypography-h1"
                  >
                    For Developers 
                  </h1>
                  <p
                    class="MuiTypography-root MuiTypography-body1"
                  >
                    Register as a developer, set up a profile, show off your skills and portfolio as they continue to grow, and get paid projects now. It's that simple, no hiring managers, pick projects and let the projects you want get matched with you today.
                  </p>
                  <div
                    class="MuiGrid-root MuiGrid-container MuiGrid-justify-xs-center"
                  >
                    <a
                      aria-disabled="false"
                      class="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-button-136 MuiButton-containedPrimary"
                      href="/register"
                      role="button"
                      tabindex="0"
                    >
                      <span
                        class="MuiButton-label"
                      >
                        <div
                          class="makeStyles-buttonText-137"
                        >
                          Get Started
                        </div>
                      </span>
                      <span
                        class="MuiTouchRipple-root"
                      />
                    </a>
                  </div>
                </div>
                <div
                  class="MuiGrid-root MuiGrid-item MuiGrid-grid-md-3 MuiGrid-grid-lg-4"
                >
                  <img
                    alt=""
                    class="makeStyles-image-173"
                    src="myapp2.svg"
                  />
                </div>
              </div>
            </div>
          </section>
          <section
            class="MuiGrid-root item-container MuiGrid-item"
          >
            <div
              class="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column"
            >
              <div
                class="MuiGrid-root makeStyles-container-174 MuiGrid-container MuiGrid-spacing-xs-3"
              >
                <div
                  class="MuiGrid-root MuiGrid-item MuiGrid-grid-md-9 MuiGrid-grid-lg-8"
                >
                  <h1
                    class="MuiTypography-root MuiTypography-h1"
                  >
                    Our Matching Process 
                  </h1>
                  <p
                    class="MuiTypography-root MuiTypography-body1"
                  >
                    PartNerdâ„¢ provides users a comprehensive recommendation generation. Get Realtime matching based on advanced data analysis.
                  </p>
                </div>
                <div
                  class="MuiGrid-root MuiGrid-item MuiGrid-grid-md-3 MuiGrid-grid-lg-4"
                >
                  <img
                    alt=""
                    class="makeStyles-image-175"
                    src="coder.svg"
                  />
                </div>
              </div>
              <img
                alt=""
                class="makeStyles-gif-176"
                src="userflow.gif"
              />
            </div>
          </section>
        </div>
      </div>
    </div>
    <style
      data-typed-js-css="true"
      type="text/css"
    >
      
        .typed-cursor{
          opacity: 1;
        }
        .typed-cursor.typed-cursor--blink{
          animation: typedjsBlink 0.7s infinite;
          -webkit-animation: typedjsBlink 0.7s infinite;
                  animation: typedjsBlink 0.7s infinite;
        }
        @keyframes typedjsBlink{
          50% { opacity: 0.0; }
        }
        @-webkit-keyframes typedjsBlink{
          0% { opacity: 1; }
          50% { opacity: 0.0; }
          100% { opacity: 1; }
        }
      
    </style>
  </body>,
  "container": <div>
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-justify-xs-center"
      style="display: none;"
    >
      <div
        class="MuiGrid-root main-container MuiGrid-container MuiGrid-direction-xs-column"
      >
        <main
          class="MuiGrid-root item-container MuiGrid-item"
        >
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3"
          >
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-md-9 MuiGrid-grid-lg-8"
            >
              <h1
                class="MuiTypography-root MuiTypography-h1"
              >
                About Us
              </h1>
              <p
                class="MuiTypography-root about MuiTypography-body1"
              >
                Welcome to PartNerd. We are a marketplace where clients and developers can form partnerships. Take the guess work out of finding a qualified developer for your existing project or next big idea and let us match you with a developer according to your needs. Our market place empowers clients to create a listing for their new or existing project and for developers to showcase their services to prospective clients.
              </p>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-md-3 MuiGrid-grid-lg-4"
            >
              <img
                alt=""
                class="makeStyles-developersImage-104"
                src="developers.svg"
              />
            </div>
          </div>
        </main>
        <section
          class="MuiGrid-root MuiGrid-item"
        >
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3 MuiGrid-direction-xs-column-reverse"
          >
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-md-3 MuiGrid-grid-lg-4"
            >
              <img
                alt=""
                class="makeStyles-image-135"
                src="reatime2.svg"
              />
            </div>
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-md-9 MuiGrid-grid-lg-8"
            >
              <h1
                class="MuiTypography-root MuiTypography-h1"
              >
                Project Owners
              </h1>
              <p
                class="MuiTypography-root MuiTypography-body1"
              >
                Find someone to build, fix, or add on to your existing site or idea. Just create an account and start searching for qualified developers in seconds. You can browse through developer profiles to to see if a developer is suited for your project or just let our algorithm do the heavy lifting for you and let us find you the the best matches based on developer skills or other preferences of your project. With Partnerd it's easy to find a developer for your project in no time.
              </p>
              <div
                class="MuiGrid-root MuiGrid-container MuiGrid-justify-xs-center"
              >
                <a
                  aria-disabled="false"
                  class="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-button-136 MuiButton-containedPrimary"
                  href="/register"
                  role="button"
                  tabindex="0"
                >
                  <span
                    class="MuiButton-label"
                  >
                    <div
                      class="makeStyles-buttonText-137"
                    >
                      Get Started
                    </div>
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </a>
              </div>
            </div>
          </div>
        </section>
        <div
          class="MuiGrid-root makeStyles-parallaxContainer-170 MuiGrid-item"
        >
          <div
            class="MuiGrid-root makeStyles-parallaxBrackround-171 MuiGrid-container MuiGrid-align-content-xs-center"
          >
            <span
              class="makeStyles-typingText-172"
            >
              <span />
              <span
                class="typed-cursor"
              >
                |
              </span>
            </span>
          </div>
        </div>
        <section
          class="MuiGrid-root item-container MuiGrid-item"
        >
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column"
          >
            <div
              class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3"
            >
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-md-9 MuiGrid-grid-lg-8"
              >
                <h1
                  class="MuiTypography-root MuiTypography-h1"
                >
                  For Developers 
                </h1>
                <p
                  class="MuiTypography-root MuiTypography-body1"
                >
                  Register as a developer, set up a profile, show off your skills and portfolio as they continue to grow, and get paid projects now. It's that simple, no hiring managers, pick projects and let the projects you want get matched with you today.
                </p>
                <div
                  class="MuiGrid-root MuiGrid-container MuiGrid-justify-xs-center"
                >
                  <a
                    aria-disabled="false"
                    class="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-button-136 MuiButton-containedPrimary"
                    href="/register"
                    role="button"
                    tabindex="0"
                  >
                    <span
                      class="MuiButton-label"
                    >
                      <div
                        class="makeStyles-buttonText-137"
                      >
                        Get Started
                      </div>
                    </span>
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </a>
                </div>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-md-3 MuiGrid-grid-lg-4"
              >
                <img
                  alt=""
                  class="makeStyles-image-173"
                  src="myapp2.svg"
                />
              </div>
            </div>
          </div>
        </section>
        <section
          class="MuiGrid-root item-container MuiGrid-item"
        >
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column"
          >
            <div
              class="MuiGrid-root makeStyles-container-174 MuiGrid-container MuiGrid-spacing-xs-3"
            >
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-md-9 MuiGrid-grid-lg-8"
              >
                <h1
                  class="MuiTypography-root MuiTypography-h1"
                >
                  Our Matching Process 
                </h1>
                <p
                  class="MuiTypography-root MuiTypography-body1"
                >
                  PartNerdâ„¢ provides users a comprehensive recommendation generation. Get Realtime matching based on advanced data analysis.
                </p>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-md-3 MuiGrid-grid-lg-4"
              >
                <img
                  alt=""
                  class="makeStyles-image-175"
                  src="coder.svg"
                />
              </div>
            </div>
            <img
              alt=""
              class="makeStyles-gif-176"
              src="userflow.gif"
            />
          </div>
        </section>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "mockSpies": Object {
    "setDarkMode": [MockFunction],
    "userDispatch": [MockFunction],
  },
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
